examples/cymbal-bank/6-appsource/cymbal-bank/ledger_cloudbuild.yaml

```diff
@@ -16,7 +16,7 @@ steps:
   - name: gcr.io/cloud-builders/gsutil
     id: download-skaffold-cache
     args: ['cp', $_CACHE_URI, '/workspace/cache'] # always write skaffold cache to filename cache in workspace
-  - name: gcr.io/k8s-skaffold/skaffold:v2.13.2
+  - name: gcr.io/k8s-skaffold/skaffold:v2.10.1
```

Comment:
Is there an issue with the newer version?

Reply:
Actually this file is now, removed... it was not being used apparently